### PR TITLE
Fix issue when creating a standard protocol application

### PR DESCRIPTION
**Pull request by RivinduM**
### Related Issues
- Issue `#1` or (None)

### Related PRs
- Related PR `#1` or (None)

### Checklist
- [ ] e2e cypress tests locally verified.
- [ ] Manual test round performed and verified.
- [ ] UX/UI review done on the final implementation.
- [ ] Documentation provided. (Add links if there are any)
- [ ] Unit tests provided. (Add links if there are any)
- [ ] Integration tests provided. (Add links if there are any)

### Security checks
- [ ] Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines?
- [ ] Ran FindSecurityBugs plugin and verified report?
- [ ] Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets?

### CODE REVIEW

File: apps/console/src/features/applications/components/wizard/minimal-application-create-wizard.tsx [hunk 1]
```diff
@@ -167,7 +167,7 @@ export const MinimalAppCreateWizard: FunctionComponent<MinimalApplicationCreateW
     const orgType: OrganizationType = useSelector((state: AppState) =>
         state?.organization?.organizationType);
     const isFAPIAppCreationEnabled: boolean = useSelector((state: AppState) =>
-        state.config.ui.features?.fapiApplicationCreation?.enabled);
+        !state.config.ui.features?.applications?.disabledFeatures?.includes("fapiApplicationCreation"));
```
Comment (brionmario) [inline apps/console/src/features/applications/components/wizard/minimal-application-create-wizard.tsx:170]: `fapiApplicationCreation` lets take this from a constant.
Check if there's already a map to store the feature ids.

REF: https://github.com/wso2/identity-apps/blob/c96eb8b986c93451340dc1f2f3bd86201f7f2c81/apps/console/src/features/groups/constants/group-constants.ts#L40

Comment (brionmario) [inline apps/console/src/features/applications/components/wizard/minimal-application-create-wizard.tsx:170]: Shall we move this to a constant in `ApplicationManagmentConstants` class.

Ref: https://github.com/wso2/identity-apps/blob/c96eb8b986c93451340dc1f2f3bd86201f7f2c81/modules/common/src/constants/organization-constants.ts#L26

Comment (RivinduM) [reply apps/console/src/features/applications/components/wizard/minimal-application-create-wizard.tsx]: fixed with [dc28d6c](https://github.com/wso2/identity-apps/pull/4795/commits/dc28d6ceb08fe37ed124f1753cb954a516cea6ab)

File: apps/console/src/features/applications/constants/application-management.ts [hunk 2]
```diff
@@ -64,7 +64,8 @@ export class ApplicationManagementConstants {
         .set("APPLICATION_EDIT_PROVISIONING_SETTINGS", "applications.edit.provisioningSettings")
         .set("APPLICATION_EDIT_ADVANCED_SETTINGS", "applications.edit.advancedSettings")
         .set("APPLICATION_SHARED_ACCESS", "applications.edit.sharedAccess")
-        .set("APPLICATION_EDIT_INFO", "applications.edit.info");
+        .set("APPLICATION_EDIT_INFO", "applications.edit.info")
+        .set("FAPI_APP_CREATION", "fapiApplicationCreation");
```
[the remark this answers is not in the document]
Comment (RivinduM) [reply apps/console/src/features/applications/constants/application-management.ts]: fixed with 967ed4d

**Review by brionmario — APPROVED** [review]

**Pull request merged**